src/languages/dsconfig.js

```diff
@@ -15,12 +15,12 @@ export default function(hljs) {
   };
   var UNQUOTED_PROPERTY = {
     className: 'string',
-    begin: '[\\w-?]+:\\w+', end: '\\W',
+    begin: /[\w\-?]+:\w+/, end: /\W/,
```

Comment:
I don't completely understand when `-` needs to be escaped inside a `[]` block since sometimes it seems to figure it out on context (or maybe it's my imagination).

Reply:
Ah, seems simple enough... in cases where it could be ambiguous it now wants it to be escaped:

```
regex = /[a\-b]/u
```

But if you just always put it at the end of the character group then it's obvious it's not a range.

Reply:
From the changes I've seen up until this file, it looks like you've moved the `-` to the end of each `[]` or it was already there, do you want to keep that consistent?

Reply:
I'm happy now that I understand it.